plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1283,6 +1290,24 @@ def construct_update_data(
             layout_traces_list.append(trace_reduced)
         return layout_traces_list
 
+    @staticmethod
+    def _parse_dtype_orjson(series: np.ndarray) -> np.ndarray:
+        """Verify the orjson compatibility of the series and convert it if needed."""
+        # NOTE:
+        #    * float16 and float128 aren't supported with latest orjson versions (3.8.1)
+        #    * this method assumes that the it will not get a float128 series
+        if series.dtype in [np.float16]:
```

Comment:
Hmmm, actually this allows for weird / unexpected behavior - see below

`y = np.randon.randn(3_000).astype(np.float16)`

When len(y) > `default_n_shown_samples` => *works like a charm* :fire: 
![image](https://user-images.githubusercontent.com/18898740/200041559-237e88e2-dc1b-4048-8908-7897fff88b2a.png)

When len(y) <= `default_n_shown_samples` => *crashes* :boom: 

![image](https://user-images.githubusercontent.com/18898740/200041961-85dbb99b-32d1-4d75-983d-8f8e909cdfd0.png)

The eventual error is quite clean tho
`TypeError: unsupported datatype in numpy array`

---

What is your opinion on this? @jonasvdd

Reply:
Maybe we better add smth to the FAQ?

Reply:
I think you've cut of the error message, will try to reproduce this locally and see how clean the error message actually is.

Reply:
okay indeed the error stacktrace is rather clean ⬇️ 

![image](https://user-images.githubusercontent.com/38005924/200127689-599950b2-7f72-4d86-9afb-4eedb50064d1.png)